src/axom/core/Macros.hpp

```diff
@@ -69,15 +69,15 @@
  * \def AXOM_LAMBDA
  *
  * \brief Convenience macro used for lambda capture by value.
- * \note When CUDA is used, the macro always expands to a host/device lambda.
+ * \note When CUDA or HIP is used, the macro always expands to a host/device lambda.
  *
- * \warning When compiling with CUDA, host/device lambdas incur a significant
+ * \warning When compiling with CUDA or HIP, host/device lambdas incur a significant
```

Comment:
The comment about performance of host execution with a host-device decorated lambda expression is no longer true. We worked with NVIDIA to come up with a better implementation. As a result, the host execution performance is nearly the same as a kernel using a lambda without the host-device decoration. So I would remove the comment.